apps/nextjs-website/react-components/components/Accordion/AccordionItem.tsx

```diff
@@ -32,14 +44,19 @@ export const AccordionItem: React.FC<AccordionItemProps> = ({
         targetItem.scrollIntoView({ behavior: 'smooth', block: 'center' });
       }
     }
-  }, [])
+  }, []);
+
+  const handleToggle = () => {
+    const newExpanded = !expanded;
+    setExpanded(newExpanded);
+    updateURL(itemID, newExpanded);
+  };
```

Comment:
```suggestion
  const handleChange = () => {
    if (!expanded) {
      appendItemIDToURLHash();
    }
    setExpanded(!expanded);
  };
```
- Change function name for consistency (onChange <--> handleChange)
- Update function to use modified function as suggested in previous review comment